### PR TITLE
fix: Orphaned form label

### PR DESCRIPTION
**Pull request by alainajaffer**
## Description

OCCURENCE 1 AND 2 DONE IN OTHER PRS

EXPECTED
example of correct use of form label:

<form>
<label for="name">Your Name:</label>
<input id="name" type="text" placeholder="Enter your Name">
<input type="submit" value="Submit"> 
</form>
ACTUAL
A form label is present, but it is not correctly associated with a form control.

OCCURENCE 1 - [Select Time Restrictions - Create Fares Data Service (dft-cfd.com)](https://preprod.dft-cfd.com/selectTimeRestrictions)
Open image-20240624-100943.png
image-20240624-100943.png

<div class="govuk-radios govuk-radios--conditional" data-module="govuk-radios"><div class="govuk-radios__item">

<input type="radio" class="govuk-radios__input" id="valid-days-required" name="timeRestrictionChoice" value="Premade" aria-controls="conditional-time-restriction" aria-expanded="false">

<label class="govuk-label govuk-radios__label" for="yes-choice">
Yes
</label>

</div>
OCCURENCE 2 - [Multiple Product - Create Fares Data Service (dft-cfd.com)](https://preprod.dft-cfd.com/multipleProducts)
Open image-20240625-172131.png
image-20240625-172131.png

<label class="govuk-label" for="product-details-period-duration">
Period duration
</label>
OCCURENCE 3 - [Search Operators - Create Fares Data Service (dft-cfd.com)](https://preprod.dft-cfd.com/searchOperators?searchOperator=ARR) && [Multiple Operators Service List - Create Fares Data Service (dft-cfd.com)](https://preprod.dft-cfd.com/multiOperatorServiceList)
Open image-20240626-093019.png
image-20240626-093019.png
Open image-20240626-112623.png
image-20240626-112623.png
<label id="operator-to-add-0" role="input" for="id" class="govuk-label govuk-checkboxes__label">
Arriva Wales

-

ACYM
</label>

SOLUTION
An incorrectly associated label does not provide functionality or information about the form control to the user. 

 

Possible causes

a <label> element

does not surround a form control and the for attribute is missing/empty

references an element that is not present in the page

references an element that is not an <input>, <select> or <textarea> element

references an <input> element with image, submit, reset, button, or hidden type

## Testing instructions

Use wave tool on page or code review

### CODE REVIEW

File: repos/fdbt-site/src/pages/searchOperators.tsx
Comment (DeannaSharma) [inline repos/fdbt-site/src/pages/searchOperators.tsx:1]: Time restriction page still has orphaned form labels:
image

Comment (DeannaSharma) [reply repos/fdbt-site/src/pages/searchOperators.tsx:1]: In the multiple operator fare  flow, when you select all services for the other operator there are still orphaned form labels:
image

Comment (DeannaSharma) [reply repos/fdbt-site/src/pages/searchOperators.tsx:1]: On the page after "select all services" there are still some orphaned form labels:
image

Comment (alainajaffer) [reply repos/fdbt-site/src/pages/searchOperators.tsx:1]: > Time restriction page still has orphaned form labels: image

https://github.com/fares-data-build-tool/create-data/pull/911 fixed in

Comment (alainajaffer) [reply repos/fdbt-site/src/pages/searchOperators.tsx:1]: > In the multiple operator fare flow, when you select all services for the other operator there are still orphaned form labels: image

was fixed in another pr

**Review by DeannaSharma — APPROVED** [review]

**Pull request merged**